packages/synchro-charts/src/testing/test-routes/widget-test-route.tsx

```diff
@@ -71,11 +80,16 @@ export class WidgetTestRoute {
       delete viewport.start;
       delete viewport.end;
     }
+    if (yMin || yMax) {
+      viewport = Object.assign(viewport, { yMin, yMax });
```

Comment:
instead of doing this, can we just add these to ln 71?

Reply:
Done